tools/MODULE.bazel

```
@@ -0,0 +1,51 @@
+# *******************************************************************************
+# Copyright (c) 2025 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# SPDX-License-Identifier: Apache-2.0
+# *******************************************************************************
+
+module(
+    name = "score_linter",
+    version = "0.1.0",
+    compatibility_level = 1,
```

This *must* match the major version. Otherwise we cannot publish it in our registry.

```suggestion
    compatibility_level = 0,
```